cartofacade/capi.go

```diff
@@ -452,6 +474,27 @@ func toIMUReading(imu string, reading s.TimedIMUReadingResponse) C.viam_carto_im
 	return sr
 }
 
+func toOdometerReading(movementSensor string, reading s.TimedOdometerReadingResponse) C.viam_carto_odometer_reading {
+	sr := C.viam_carto_odometer_reading{}
+	sensorCStr := C.CString(movementSensor)
+	defer C.free(unsafe.Pointer(sensorCStr))
+	sr.odometer = C.blk2bstr(unsafe.Pointer(sensorCStr), C.int(len(movementSensor)))
+
+	translation := spatialmath.GeoPointToPose(reading.Position, geo.NewPoint(0, 0)).Point()
```

Comment:
I don't think this function is in latest anymore. But the logic in GeoPointToPoint is what you want.